distribution/docker/solr/src/main/docker/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN unzip /jars/solr-schema-${project.version}.jar -d /prep
 # Final Image Stage
 FROM solr:${solr.version}
 # Copy solr libs
-COPY maven/jts-core-1.15.0.jar /opt/solr/server/lib/
+COPY maven/jts-core-1.15.0.jar /opt/solr/server/solr-webapp/webapp/WEB-INF/lib/
```

Comment:
See https://lucene.apache.org/solr/guide/7_7/spatial-search.html#jts-and-polygons-flat